x86_symbolic/src/Data/Macaw/X86/Symbolic.hs

```diff
@@ -472,6 +480,102 @@ instance GenArchInfo LLVMMemory M.X86_64 where
     , updateReg = x86UpdateReg
     }
 
+-- | Helper, not exported
+writeBytesBackwards ::
+  MM.HasPtrWidth w =>
+  C.IsSymBackend sym bak =>
+  (?memOpts :: MM.MemOptions) =>
+  MM.HasLLVMAnn sym =>
+  bak ->
+  MM.MemImpl sym ->
+  MM.LLVMPtr sym w ->
+  Seq.Seq Word8 ->
+  IO (MM.LLVMPtr sym w, MM.MemImpl sym)
+writeBytesBackwards bak mem ptr bytes = do
+  let sym = C.backendGetSym bak
+  let i8 = MM.bitvectorType (Bytes.toBytes (1 :: Int))
```

Comment:
It might be worth factoring out `i8 :: StorageType` into its own helper definition, as it's used both here and in `x86SysvStack`.

Reply:
No longer relevant.